vispy/glsl/misc/spatial-filters.frag

```diff
@@ -6,6 +6,15 @@ const float kernel_bias  = -0.234377;
 const float kernel_scale = 1.241974;
 uniform sampler2D u_kernel;
 
+float unpack(vec4 rgba) {
+    //return rgba.r;
+    rgba.rgba = rgba.abgr * 255;
+    float sign = 1.0 - step(128.0,rgba[0])*2.0;
+    float exponent = 2.0 * mod(rgba[0],128.0) + step(128.0,rgba[1]) - 127.0;
+    float mantissa = mod(rgba[1],128.0)*65536.0 + rgba[2]*256.0 + rgba[3] + float(0x800000);
+    return sign * exp2(exponent) * (mantissa * exp2(-23.0 ));
```

Comment:
Yeah, it doesn't look like your test image is the result of glitchy values, but something more systematic